kotlin-js/src/main/kotlin/kotlinx/js/Module.kt

```diff
@@ -0,0 +1,5 @@
+package kotlinx.js
+
+external interface Module<out T : Any> {
+    val default: T
```

Comment:
Technically default export can be null

Reply:
I see following use cases:
1. Module with default export -> use `Module`
2. Module without default export -> don't use `Module`